plugins/katello/3.16/user_guide/tracer/index.md

```diff
@@ -0,0 +1,167 @@
+---
+title: Tracer
+---
+# Tracer
+
+Tracer displays a list of services and applications that are outdated and need to be restarted.
+
+## Using Tracer
+
+- [Installing Tracer](/plugins/katello/{{ page.version }}/user_guide/tracer/index.html#installing-tracer)
+- [Viewing traces for a content host in the UI](/plugins/katello/{{ page.version }}/user_guide/tracer/index.html#viewing-traces-for-a-content-host-in-the-UI)
+- [Viewing traces for a host with the CLI](/plugins/katello/{{ page.version }}/user_guide/tracer/index.html#viewing-traces-for-a-host-with-the-CLI)
+- [Resolving traces for a single content host](/plugins/katello/{{ page.version }}/user_guide/tracer/index.html#resolving-traces-for-a-single-content-host)
+- [Resolving traces on multiple content hosts](/plugins/katello/{{ page.version }}/user_guide/tracer/index.html#resolving-traces-on-multiple-content-hosts)
+
+### Installing Tracer
+
+#### With the CLI
+
+Make sure that the "Katello Client Release Package" repository is [installed and enabled](/plugins/katello/{{ page.version }}/installation/clients.html), then install the `katello-host-tools-tracer` package:
```

Comment:
Where does this happen? On Katello or the host? It would be good to specify even if it seems obvious